src/controllers/audience/audienceController.ts

```diff
@@ -4,13 +4,15 @@ import {LearningCatalogue} from '../../learning-catalogue'
 import {Audience} from '../../learning-catalogue/model/audience'
```

Comment:
Not in changes, but line 106 redirects to 'audience-type', looks as if it should be either 'add-organisation' or 'audiences/type'